### DISCUSSION TITLE
[BUG] Taskbar weather

### DISCUSSION THREAD

**Discussion by hiranokite** [valinet/ExplorerPatcher#1041] [General]
The new option to move widget to left / top, align everything on my taskbar to the right.
is this a bug or is it something specific on my system?

![ss](https://user-images.githubusercontent.com/5361459/157551512-2e5240c8-4ebc-4114-a92b-92dcaecac150.png)

**Comment by valinet** [comment]
It's a bug, I just fixed it, will push an update very soon. Thanks for letting me know.

**Reply by hiranokite** [reply]
Thank you =)

**Reply by hiranokite** [reply]
bugs on version [22000.556.42.33]

1. if left / top is set, when restarting explorer the widget will not appear, you need to disable and enable it again, or change the widget position so that it appears from the left again.
2. if taskbar alignment is set to Centered with start button, changing widget from left / top to right / bottom cause this on taskbar
![ss](https://user-images.githubusercontent.com/5361459/157571917-db412544-6d24-4e3f-bdc9-cd6977ad1e53.png)

**Reply by valinet** [reply]
Reworked the logic, solved a few other quirks, seems to work now on fresh installs. Pushed a new pre-release (42.34).

**Reply by hiranokite** [reply]
nice, everything works fine now
